### DISCUSSION TITLE
Curation Report - Week of March 28

### DISCUSSION THREAD

**Discussion by AlexanderPico** [wikipathways/GPML-Repo#52] [Curation reports]
Hi all,

Here what happened at WikiPathways this week:

- 15 recent changes, including:
  - 3 newly approved: https://www.wikipathways.org/index.php/Pathway:WP5209, https://www.wikipathways.org/index.php/Pathway:WP5218, https://www.wikipathways.org/index.php/Pathway:WP5200
  - a test pathway: deleted
  - and lots of minor edits
- 8 under construction, including:
  - 2 newly approved: https://www.wikipathways.org/index.php/Pathway:WP5154, https://www.wikipathways.org/index.php/Pathway:WP5199 
  - 3 test pathways: deleted
 - 2 additional test pathways: deleted
 - Update a few tags
 - Tested search: works!

Additional note: trying to edit a description live on the old site still causes a major stall of the site for 10 minutes. Ugh. Editing the GPML directly (via Edit tab) works better. Can't wait to be working with the new infrastructure!

Next up: Anders!

**Comment by ariutta** [comment]
Hi all,

I had to do my curation early this week, but I see several good changes.

WP5220 <https://www.wikipathways.org/index.php/Pathway:WP5220> looks ready
to be approved, but it's still in the process of being edited, so I didn't
approve it yet.

WP5187 <https://www.wikipathways.org/index.php/Pathway:WP5187>, WP5154
<https://www.wikipathways.org/index.php/Pathway:WP5154>, WP5218
<https://www.wikipathways.org/index.php/Pathway:WP5218>: listed as approved
but also marked with "fix interactions" and "annotate nodes".

Added a few ontology annotations.

Search works.

Next up: Marvin!

-Anders

On Fri, Apr 1, 2022 at 3:39 PM Alexander Pico ***@***.***>
wrote:

> Hi all,
>
> Here what happened at WikiPathways this week:
>
>    - 15 recent changes, including:
>       - 3 newly approved:
>       https://www.wikipathways.org/index.php/Pathway:WP5209,
>       https://www.wikipathways.org/index.php/Pathway:WP5218,
>       https://www.wikipathways.org/index.php/Pathway:WP5200
>       - a test pathway: deleted
>       - and lots of minor edits
>    - 8 under construction, including:
>       - 2 newly approved:
>       https://www.wikipathways.org/index.php/Pathway:WP5154,
>       https://www.wikipathways.org/index.php/Pathway:WP5199
>       - 3 test pathways: deleted
>    - 2 additional test pathways: deleted
>    - Update a few tags
>    - Tested search: works!
>
> Additional note: trying to edit a description live on the old site still
> causes a major stall of the site for 10 minutes. Ugh. Editing the GPML
> directly (via Edit tab) works better. Can't wait to be working with the new
> infrastructure!
>
> Next up: Anders!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wikipathways/wikipathways-development/discussions/52>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJ4TA3CXFGM6NRICBFOL6LVC53KFANCNFSM5SJ6MFLA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Anders Riutta
***@***.*** | Bioinformatics Core | Institute of Data
Science and Biotechnology
Gladstone Institutes | *Science Overcoming Disease*
